src/components/ArticleDisplay.jsx

```diff
@@ -21,6 +26,38 @@ const ArticleDisplay = ({ article_id }) => {
       });
   }, []);
 
+  const upVote = (article_id) => {
+    addArticleVote(article_id)
+      .then(() => {
+        setIsUpVoteError(false);
+      })
+      .catch(() => {
+        setIsUpVoteError(true);
```

Comment:
So glad you have this, love the user has feedback. I'd like it to be clearer and have more prominence on the page. When offline the vote counter does go up before this message. It would be great for the number to not change when that happens.